src/auth/controllers/auth.controller.spec.ts

```diff
@@ -58,4 +66,44 @@ describe('AuthController', () => {
       expect(await authController.login(reqObject, loginInputDto)).toBe(null);
     });
   });
+
+  describe('refreshToken', () => {
+    let mockTokenUser: TokenUserIdentity;
```

Comment:
@asad-mlbd Just a small observation, since these are just input consts, do we need to prefix them with `mock...`? 
Not an issue but was thinking if we need it. :thinking:

Reply:
removed 'mock..'